cli/src/command/account.rs

```diff
@@ -920,81 +965,108 @@ pub async fn voting_output_command(account: &Account) -> Result<(), Error> {
-                }
+    let hrp = address.hrp();
+
+    for output_data in unspent_output_ids.iter() {
+        let output_id = &output_data.output_id;
+
+        outputs.push((*output_id, output_data.output.kind_str().to_string()));
+
+        // Output might be associated with the address, but can't be unlocked by it, so we check that here.
+        // Panic: cannot fail for outputs belonging to an account.
+        let (required_address, _) = output_data
+            .output
+            .required_and_unlocked_address(current_time, output_id, None)
+            .unwrap();
+
+        if address.inner() == &required_address {
+            if let Some(nts) = output_data.output.native_tokens() {
+                native_tokens.add_native_tokens(nts.clone())?;
+            }
+            match &output_data.output {
+                Output::Nft(nft) => nfts.push((
+                    nft.nft_id_non_null(output_id),
+                    nft.nft_address(output_id).to_bech32(*hrp),
+                )),
+                Output::Alias(alias) => aliases.push((
+                    alias.alias_id_non_null(output_id),
+                    alias.alias_address(output_id).to_bech32(*hrp),
+                )),
+                Output::Basic(_) | Output::Foundry(_) | Output::Treasury(_) => {}
             }
+            let unlock_conditions = output_data
+                .output
+                .unlock_conditions()
+                .expect("output must have unlock conditions");
+            let sdr_amount = unlock_conditions
+                .storage_deposit_return()
+                .map(|sdr| sdr.amount())
+                .unwrap_or(0);
+
+            amount += output_data.output.amount() - sdr_amount;
         }
     }
 
-    log = format!(
-        "{log}\nOutputs: {:#?}\nBase coin amount: {}\nNative Tokens: {:#?}\nNFTs: {:#?}\nAliases: {:#?}\nFoundries: {:#?}\n",
-        output_ids,
-        amount,
-        native_tokens.finish_vec()?,
-        nfts,
-        aliases,
-        foundries,
-    );
+    formatted_string.push_str("\nOutputs:");
+    for (id, kind) in outputs {
+        formatted_string.push_str(&format!("{0}{1:<6}{id}{0}{2:<6}{kind}", "\n  ", "Id:", "Type:"));
+    }
+    formatted_string.push_str("\n");
+
+    formatted_string.push_str("\nBase Tokens:");
+    formatted_string.push_str(&format!("{0}{1:<8}{2}\n", "\n  ", "Amount:", amount));
+
+    formatted_string.push_str("\nNative Tokens:");
+    for (id, amount) in native_tokens
+        .finish_vec()?
+        .into_iter()
+        .map(|nt| (*nt.token_id(), nt.amount().to_string()))
+    {
+        formatted_string.push_str(&format!("{0}{1:<8}{id}{0}{2:<8}{amount}", "\n  ", "Id:", "Amount:"));
+    }
+    formatted_string.push_str("\n");
+
+    formatted_string.push_str("\nNFTs:");
+    for (id, bech32) in nfts.into_iter() {
+        formatted_string.push_str(&format!("{0}{1:<8}{id}{0}{2:<8}{bech32}", "\n  ", "Id:", "Bech32:"));
```

Comment:
I wouldn't print the Bech32 of NFT/Alias here tbh, not very relevant

Reply:
just to make sure ... this correct the bech32 address, tho, right? But you don't want to see it in the details list, but rather in the `addresses` list?

Reply:
Yes I think this is redundant information that shouldn't be displayed here

Reply:
forgot to say it ... done for some time now